Source/WebCore/Modules/webaudio/AudioContext.cpp

```diff
@@ -187,6 +189,20 @@ double AudioContext::baseLatency()
     return static_cast<double>(destination().framesPerBuffer()) / sampleRate();
 }
 
+double AudioContext::outputLatency()
```

Comment:
It seems odd to return the audiosession latency even if AudioContext is not playing.
I'll tend to move the logic to add the latencies in destination.

Reply:
Limited it to when the context is playing